### PR TITLE
30 make building model

### PR DESCRIPTION
**Pull request by aaaustin10**

### CODE REVIEW

File: exdb/tests/integration_tests.py [hunk 1]
```diff
@@ -71,9 +74,9 @@ def test_type_str_method(self):
         self.assertEqual(str(Type.objects.get(pk=t.pk)), t.name, "Type object should have been created.")
 
     def test_organization_str_method(self):
```
Comment (tom08) [inline exdb/tests/integration_tests.py]: Should we rename this method to reflect the method being tested?

**Comment by shaneallgeier** [comment]
Please update this with master

**Comment by shaneallgeier** [comment]
I approve.

**Comment by enki211** [comment]
Looks good

**Comment by tom08** [comment]
If we decided not to rename that method, I approve.

**Pull request merged**